coss/opensource_clubs/templates/opensource_clubs/category_landing_page.html

```diff
@@ -1,33 +1,24 @@
 {% extends "opensource_clubs/category_base.html" %}
 {% load static compress wagtailcore_tags wagtailimages_tags %}
 
-{% compress css %}
-  {% block extra_css %}
-    {{ block.super }}
-    <link rel="stylesheet" type="text/scss" href="{% static 'css/entity.scss' %}">
-  {% endblock %}
-{% endcompress %}
-
-{% block body_class %}entity{% endblock %}
+{% block header_class %}white-header{% endblock %}
 
 {% block jumbotron %}
   <div class="container jumbotron">
     <div class="row">
-      <div class="col-md-12">
-        <h2>
-          {{ page.title }}
-        </h2>
-      </div>
+      {% include 'home/blocks/category_heading.html' with heading=page.heading %}
     </div>
   </div>
 {% endblock %}
 
 {% block content %}
-  <div class="container content">
-    <div class="card-deck">
-      {% for club in page.get_children %}
-        {% include 'opensource_clubs/club_card.html' with club=club.specific %}
-      {% endfor %}
+  <div class="content inverted">
```

Comment:
This should be wrapped in an `if` block in case there are no children.

Reply:
Maybe it's a good idea to have an `else` clause too, to display something. Maybe it could something simple like a text paragraph that can guide the content creators on how to add content for this page. Something along the lines of " You need to add entities too.". Thoughts?